media/js/pocket/mobile-nav-init.es6.js

```diff
@@ -4,6 +4,16 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
  */
 
-import init from './mobile-nav.es6';
+import { init } from './mobile-nav.es6';
+
+// holder for timeout
+let timeout = false;
 
 init();
+// debounced resize event listener
+window.addEventListener('resize', function () {
```

Comment:
Instead of using a `resize` event, perhaps initializing the nav only for smaller screens using `matchMedia.addListener()` might be more reliable? https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList/addListener

Reply:
Although, one thing I haven't quite figured out yet is why `resize` is firing on page load to begin with. Is this normal? 🤔

Reply:
Looks good but I'm wary of the deprecation warning

Reply:
the "change" event on a media query seems active though: https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryListEvent

Reply:
Ah yeah, I keep forgetting they replaced it with https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList/change_event